app/views/career_site/positions/show.html.slim

```diff
@@ -68,4 +68,4 @@
           size: :medium,
           class: "flex-grow-1 flex-md-grow-0 mt-2" \
         ).with_content(t("career_site.positions.submit_application"))
-      .mt-2.font-small.text-gray-600 = t("recaptcha.privacy_message_html")
+      .mt-2.font-small.text-secondary = t("recaptcha.privacy_message_html")
```

Comment:
may we use `tblr-gray-500` ?

Reply:
as i see `text-secondary` hangs not `tblr-gray-500`

Reply:
text-secondary is gray-500

Reply:
And there's no text-gray-500 class